.buildkite/x-pack/pipeline.xpack.auditbeat.yml

```diff
@@ -406,3 +406,24 @@ steps:
         notify:
           - github_commit_status:
               context: "x-pack/auditbeat: Packaging Linux arm64"
+
+      - label: ":ubuntu: x-pack/auditbeat: Packaging Linux FIPS"
+        key: "packaging-linux-fips"
+        env:
+          PLATFORMS: "+all linux/amd64 linux/arm64"
```

Comment:
is this expected? I see there is a mix for arm and and, but I also see some top-level env variables for each of those platforms:

```
  PLATFORMS_AMD64_FIPS: "+all linux/amd64"
  PLATFORMS_ARM_FIPS: "+all linux/arm64"
```


What's the reason for this?

Reply:
@v1v when yo say that we have some top-level env variables are you referring to the ones defined in `.buildkite/packaging.pipeline.yml` ? 
This pipeline runs for PRs and main/release branches commit depending on changes under the related directory using the monorepo plugin, while `.buildkite/packaging.pipeline.yml` is run only on main and release branches to generate Beats DRAs.

We can define Env vars containing the platforms for these smaller pipeline as well, what do you think @pazone ?

Reply:
> when you say that we have some top-level env variables are you referring to the ones defined in .buildkite/packaging.pipeline.yml ?

Yes, thanks for double checking

Reply:
Added PLATFORMS_*_FIPS in  394149f7e2f796040aef963390df1d7a437c5f21